doc/manuals/devel/HttpHeaders.md

```diff
@@ -0,0 +1,354 @@
+List of HTTP Headers used by Orion
```

Comment:
File name should use the camelCase format, i.e. to be `httpHeaders.md`.

In addition, it should be linked from doc/manuals/devel/README.md#contents (maybe between "Semaphores" and "Cookbook").